### ISSUE TITLE
IRIS Q&As: Education -- Section 702

### ISSUE THREAD

**Issue by lwalsh617** [department-of-veterans-affairs/va.gov-team#20644]
##Issue
For the launch of Resources and support MVP 1.1, content that fall into the "Section 702" category under education & training needs to be developed based on the following:

https://gibill.custhelp.va.gov/
Product guide pages
Credible external sources as needed
Content will take the form of a Word document, but will be organized and labeled so that, when final it, it can be easily flowed into learning center page templates currently in Drupal.

Content will meet  [content component checklist](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsa/teams/sitewide-content/Content%20Components%20Check%20List%20(2).docx) requirements.

##Tasks
- [x] PW to identify IRIS faqs to migrate (Laura)

- [x]  PW to identify any content in the Contact Center that should inform editing of IRIS faqs, or be added as additional content (Laura)

- [ ] PW to develop draft copy (Laura)

- [ ] PW to assign tags (Laura)

- [ ] DEPO to complete initial review (Beth)

- [ ] PW to copyedit draft (Randi)

- [ ] PW to create content review GH ticket and attach edited draft for DEPO review (Laura)

##Acceptance Criteria
- [ ] Copy deck ready for 2nd DEPO review and attached to a content review GH ticket.

##Most recent copy deck
Please use the deck attached and replace it when you make edits/comments to ensure that the latest document is always here, at the top of this ticket.
Copy deck version:

**Comment by lwalsh617** [comment]
Noting here that once this article is published, we should link to it from R&S coverage article Tuition and fees the GI Bill Covers under the section GI Bill benefits for in-state tuition

**Comment by lwalsh617** [comment]
@bethpottsVADEPO -- Here's the preview node to the revised article: http://preview-prod.vfs.va.gov/preview?nodeId=37717

**Comment by lwalsh617** [comment]
@bethpottsVADEPO - If you drop your feedback here, I'll revise for this sprint. Thanks

**Comment by bethpottsVADEPO** [comment]
@lwalsh617 Here's some specific feedback for you to work with, then please ask @RLHecht to copyedit, then pass back to me:

- H1: I see 0 MSV for the current H1. I'd like to discuss whether "section 702" should be a part of the H1. I'd like to consider: In-state tuition rates under the Veterans Choice Act, or In-state tuition rates under Section 702 of the Veterans Choice Act.
- Would you shift the focus from "section 702" to "instate tuition" throughout--both in the H2s and in running text? You don't need to avoid using the term "section 702" altogether, but since the content in this article is about how you can get in-state tuition, can you focus it in a more PL way on that aspect of the topic?
- **The first H2** could be more specific, like "The difference between in-state and out-of-state tuition rates." Then for the text, aren't all Veterans covered under Section 702? It's just that not all schools participate, I think. So that first sentence needs to be revised--and also shortened. Same for the last sentence, it just needs clarification: If the school is getting GI Bill funds, they need to comply with Section 702. Maybe that last sentence could say something like, "But if a school gets GI Bill funds, as a Veteran you don't need to meet these same requirements." (By "covered individuals," did you mean Veterans? There may be something I'm not clear on here, but if we can simplify that phrasing to "you," let's do it.) And if this rewrite makes sense, you'll also want to change the previous sentence to "Many schools require students to prove they've lived in the state..."
- **Second H2:** With the first H2's text revisions above, make sure there isn't repetition in the first paragraph here. For the second paragraph, it's the schools not the states that offer in-state tuition, so could you rephrase that? Couple PL edits to make: obligated/ensure/comply/issue payments. And unbold "any."
- Unbold "or." Change "you first have to receive benefits" to "you need to be receiving benefits"
- What does this mean?: Section 702 doesn't apply to other VA programs like VR&E. (Do we mean that the Veterans doesn't automatically also qualify for VR&E?) Could we remove this sentence? Or place it somewhere else on the page? It seems a little out of context here. 
- This isn't our voice: The law defines a "covered individual" as:--please rephrase that.
- The organization of the info in the first bullet needs some reworking. The ways to prove your discharge date needs bullets, and since we generally avoid subbullets, can you try an H3 of "Who qualifies as a "covered individual." (There may be a standard already for how to say this, and I think it's worth looking around our health care pages for how we talk about groups of people who are covered.) Then try H4s for the 3 types of covered individuals. Again, I think there's precedent for this on some HC pages. Thank you for trying to add some structure to this section.
- Simplify/PL the first subhead (not sure which level this will be after you do the above work) to something like: How to keep your status as a covered individual. I also don't think we need to keep using the quotation marks for this term in running text.
- The next subhead could be simplified to "Residency requirements." And could you try reworking the lead-in to the bullets as well as the text so that the repetition is eliminated and we present both "you must live in the state where the school is located" and "in order to continue getting in-state tuition" only once? The additional potential requirements to call out are: show intent and become a legal resident within a certain period of time.
- The next subhead "Billing errors" could be more PL, something like this: "What to do if your school charges you out-of-state tuition by mistake" (you may have to shorten that). Also there's no need to say "to VA"--it can either be "to us" or just "submit a complaint."
- **Third H2:** Could this section be combined with the very first section? They are both about the differences between in-state and out-of-state. I'm not sure if in the larger context of this article it'll make sense to have it lead off the page or wrap up the page--can you play with the placement and see what works best?

**Comment by bethpottsVADEPO** [comment]
@lwalsh617 I wanted to check in on this and see where you were with it.

**Comment by lwalsh617** [comment]
@bethpottsVADEPO - I haven't been able to access CAG since Tuesday when my contract was terminated so am unable to wrap this up. I'm meeting with Randi and Selina before the 31st to pass things off so will let Randi know she can copyedit and share the node.

**Comment by bethpottsVADEPO** [comment]
@lwalsh617 Thanks so much.

**Comment by bethpottsVADEPO** [comment]
@RLHecht Would you please copyedit this before I send the preview link to SMEs?
http://preview-prod.vfs.va.gov/preview?nodeId=37717

After you copyedit, you can close this ticket and open a new one for stakeholder approval and publishing. See above in this ticket for crosslinking notes to carry over to that new ticket. Also consider where we should crosslink to this article from tier 1 pages. Thanks!

**Comment by RLHecht** [comment]
@bethpottsVADEPO I've copyedited this and it's ready to send to stakeholders. I'll open up the new ticket. Closing this one.
